rust/src/shards/gui/containers/panels.rs

```diff
@@ -124,6 +124,16 @@ macro_rules! impl_panel {
         Some(&self.requiring)
       }
 
+      fn exposedVariables(&mut self) -> Option<&ExposedTypes> {
```

Comment:
Panels are always visible and unconditionals so I think it should work.

Reply:
👍

Reply:
Would that have the same effect completely?

Reply:
They would at least be able to setup variables before the first UI element. Something like:

```clj
(UI.Window
  (-> (Setup false >= .myBool)
     (UI.Collapsing ...)
  ))
```

Reply:
I mean UI wise, appearance wise

Reply:
No really, it does look different.

In the first case, the Window is collapsible at the title bar level.
![image](https://user-images.githubusercontent.com/3006525/186102839-6bccdc8a-6494-4111-a534-4f40a4c2a01e.png)

In the second case it would be the content (with optionally an empty header).
![image](https://user-images.githubusercontent.com/3006525/186103189-45fe361b-9d36-4fc4-a7dd-aa3727eef99b.png)

Reply:
I like the first case, Window doesn't need to expose variables. it's fine really